docs/source/dashboard.rst

```diff
@@ -0,0 +1,135 @@
+Dashboard Diagnostics
+=====================
+
+Profiling parallel code can be challenging, but the :doc:`Dask distributed scheduler <scheduling>` 
+provides live feedback via its interactive dashboard. 
+
+## Include dashboard picture status page
+
+A link that redirects to the dashboard will prompt in the terminal where the scheduler is
+created, and it is also shown when you create a ``Client`` and connect the scheduler.
+
+.. code-block:: python
+
+   from dask.distributed import Client
+   client = Client()  # start distributed scheduler locally. 
+   client            
+
+In a Jupyter Notebook or Jupyter Lab session displaying the client object will show the dashboard address
+as following.
+
+.. figure:: images/dashboard_link.png
+    :alt: NEEDS ALT TEXT
+
+The address of the dashboard will be displayed if you are in a Jupyter Notebook,
+or it can be queried from ``client.dashboard_link``. By default, when starting a scheduler 
+n your local machine the dashboard will be served at ``http://localhost:8787/status``, you
+can type this address into your browser to access the dashboard but may be served 
+elsewhere if this port is taken. You can also configure the address by passing options to the 
+scheduler, see ``dashboard_address`` in `LocalCluster < https://docs.dask.org/en/stable/deploying-python.html#reference>`_
+
+The dashboard link redirects you to main dashboard page as shown below:
+
+.. figure:: images/dashboard_status.png
+    :alt: NEEDS ALT TEXT
+
+In the entry point of the dashboard you can find multiple plots with information about your cluster:
+
+- Bytes stored: Cluster memory. 
```

Comment:
This page is likely to get long, I think it would make sense to have this list serve as a bit of a table of contents, and add anchors for ease of navigation